python/primihub/FL/model/logistic_regression/homo_lr.py

```diff
@@ -56,39 +56,38 @@ def set_theta(self, theta):
             theta = np.array(theta)
         self.theta = theta
```

Comment:
The feature name should be kept for interrencing.

Reply:
OK